pipelines/ocw/scripts/pipeline-ocw-www.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+SITE_BRANCH=$1  # Should be "preview" or "release"
+if [[ $SITE_BRANCH == "preview" ]]
+then
+  FASTLY_SERVICE_ID=$FASTLY_SERVICE_ID_PREVIEW
+  FASTLY_API_TOKEN=$FASTLY_API_TOKEN_PREVIEW
+  OCW_CONTENT_BUCKET=$OCW_CONTENT_BUCKET_PREVIEW
+elif [[ $SITE_BRANCH == "release" ]]
+then
+  FASTLY_SERVICE_ID=$FASTLY_SERVICE_ID_RELEASE
+  FASTLY_API_TOKEN=$FASTLY_API_TOKEN_RELEASE
+  OCW_CONTENT_BUCKET=$OCW_CONTENT_BUCKET_RELEASE
+else
+  echo "Invalid branch $1"
+  exit 1
+fi
+
+yes | fly -t $FLY_TARGET set-pipeline \
+-p ocw-www-pipeline-$SITE_BRANCH \
+--team=$FLY_TEAM \
+--config=pipelines/ocw/pipeline-ocw-www.yml \
+-v artifact-bucket=$ARTIFACT_BUCKET \
+-v git-domain=$GITHUB_DOMAIN \
+-v github-org=$GIT_CONFIG_ORG \
+-v hugo-theme-branch=$HUGO_THEME_BRANCH \
+-v ocw-www-repo=$OCW_WWW_REPO \
+-v ocw-www-repo-branch=$SITE_BRANCH \
+-v ocw-bucket=$OCW_CONTENT_BUCKET \
+-v ocw-studio-url=$OCW_STUDIO_BASE_URL \
+-v fastly-service-id=$FASTLY_SERVICE_ID \
+-v fastly-api-token=$FASTLY_API_TOKEN \
```

Comment:
This should be getting pulled from Vault at runtime, so we shouldn't be setting it via the CLI

Reply:
I was assuming all these would already be set as environment variables, via vault or some other process.  How would I specify that these values come from vault directly instead of an env. variable?

Reply:
If all these vars are defined in vault (`ocw-bucket`, `fastly-service-id`, etc), will concourse automatically find them assuming `CONCOURSE_VAULT_URL` is set?

Reply:
Related comment:
> it would probably be best to create a few files that have all of the correct values for a given destination

I could use some guidance on how to best handle this.  Some of these values should not be public (fastly api token, github key), so should I create a file for values that are okay to be public but the rest should go in vault?  And if they are in vault, does anything need to be modified in the pipeline generation script to get the values from there?

Reply:
Sorry, thought I had already responded to this. So, any unset var in the pipeline will automatically attempt to look up a key by that name in the configured credential provider (in this case Vault). More details of how that works are available at https://concourse-ci.org/vault-credential-manager.html

So, in order to handle those vars we just need to set the values in Vault. For values from Vault that have multiple values (e.g. Fastly API token and service Id) I like to keep those as a single key in Vault so that the var would look like `fastly.api_token` and `fastly.service_id`. The different values for preview and published Fastly distributions would be located based on the name of the pipeline group, so there's no need to special case the var names in the pipeline.

Reply:
> on the name of the pipeline group

There aren't any pipeline groups specified yet, so I'll work on adding that too.